File: adobe/android/src/main/java/com/theoplayer/reactnative/adobe/AdobeEvent.kt
package com.theoplayer.reactnative.adobe

enum class AdobeEventTypes(val value: String) {
  SESSION_START("sessionStart"),
  PLAY("play"),
  PING("ping"),
  BITRATE_CHANGE("bitrateChange"),
  BUFFER_START("bufferStart"),
  PAUSE_START("pauseStart"),
  AD_BREAK_START("adBreakStart"),
  AD_START("adStart"),
  AD_COMPLETE("adComplete"),
  AD_SKIP("adSkip"),
  AD_BREAK_COMPLETE("adBreakComplete"),
  CHAPTER_START("chapterStart"),
  CHAPTER_SKIP("chapterSkip"),
  CHAPTER_COMPLETE("chapterComplete"),
  ERROR("error"),
  SESSION_END("sessionEnd"),
  SESSION_COMPLETE("sessionComplete")
}

File: adobe/ios/Connector/AdobeEvent.swift
// AdobeEvent.swift

enum AdobeEventTypes: String {
    case SESSION_START = "sessionStart"
    case PLAY = "play"
    case PING = "ping"
    case BITRATE_CHANGE = "bitrateChange"
    case BUFFER_START = "bufferStart"
    case PAUSE_START = "pauseStart"
    case AD_BREAK_START = "adBreakStart"
    case AD_START = "adStart"
    case AD_COMPLETE = "adComplete"
    case AD_SKIP = "adSkip"
    case AD_BREAK_COMPLETE = "adBreakComplete"
    case CHAPTER_START = "chapterStart"
    case CHAPTER_SKIP = "chapterSkip"
    case CHAPTER_COMPLETE = "chapterComplete"
    case ERROR = "error"
    case SESSION_END = "sessionEnd"
    case SESSION_COMPLETE = "sessionComplete"
}
